charts/dapr/crds/configuration.yaml

```diff
@@ -284,7 +284,7 @@ spec:
                     - protocol
                     type: object
                   samplingRate:
-                    type: string
```

Comment:
I'd prefer not to change this and maintain backwards-compatibility here. Changing this CRD will cause a lot of headaches to people running Dapr in production already.